src/include/apparmor/profile_dialogs.rb

```diff
@@ -1477,19 +1477,11 @@ def SelectProfileForm(profiles, formhelp, formtitle, iconname)
             next
           end # TODO ELSE POPUP NO ENTRY SELECTED ERROR
         elsif id == :delete
-          if Popup.YesNoHeadline(
-              _("Delete profile confirmation"),
-              Ops.add(
-                Ops.add(
-                  _("Are you sure you want to delete the profile "),
-                  profilename
-                ),
-                _(
-                  " ?\nAfter this operation the AppArmor module will reload the profile set."
-                )
-              )
-            )
-            Builtins.y2milestone(Ops.add("Deleted ", profilename))
+          # Translators: %1 is the name of the profile.
+          popup_msg = Builtins.sformat(_("Are you sure you want to delete the profile \"%1\"?"), profilename )
```

Comment:
I do not know how the profile names are usually long, but in some languages the translation might be quite long. I'd prefer to add a line break there, just in case...

```suggestion
          popup_msg = Builtins.sformat(_("Are you sure you want to delete the profile\n\"%1\"?"), profilename )
```

Reply:
Fine with me.